state/action.go

```diff
@@ -112,8 +112,18 @@ type actionDoc struct {
 
 // ActionMessage represents a progress message logged by an action.
 type ActionMessage struct {
-	Message   string    `bson:"message"`
-	Timestamp time.Time `bson:"timestamp"`
+	Message_   string    `bson:"message"`
```

Comment:
This just seems a bit clumsy. Possibly MessageValue and TimestampValue

Reply:
We use this elsewhere when we need to have stuff available for serialisation but it's really internal (like in the juju/description package). I think this is the first time in juju/juju so maybe best to avoid it here. I can change to Value